### ISSUE TITLE
https://angular.io/guide/security#enforcing-trusted-types

### ISSUE THREAD

**Issue by tiddlypip** [angular/angular#45438]
### Description

I've found that Angular 13 now also requires angular#bundler as a trusted type when aot is enabled, angular#unsafe-jit is only required when aot compilation is turned off. The documentation misses off the required type angular#bundler and only found by searching source code etc.

### What is the affected URL?

https://angular.io/guide/security#enforcing-trusted-types

### Please provide the steps to reproduce the issue

Add content security policy with trusted types specified for project with aot turned on. On running hosted application the application is blocked by CSP. Adding angular#bundler to the trusted types fixes the issue.

### Please provide the expected behavior vs the actual behavior you encountered

The documentation should include angular#bundler in the list of trusted types and distinguish between the trusted types for aot and jit compilation.

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

```true
Its specific to chrome, firefox currently ignores trusted types.
```

**Comment by alan-agius4** [comment]
Duplicate of https://github.com/angular/angular/issues/44647
